services-turf/src/main/java/com/mapbox/turf/TurfAssertions.java

```diff
@@ -19,21 +18,6 @@ private TurfAssertions() {
     // Private constructor preventing initialization of this class
   }
 
-  /**
-   * Unwrap a coordinate {@link Point} from a Feature with a Point geometry.
-   *
-   * @param obj any value
-   * @return a coordinate
-   * @see <a href="http://turfjs.org/docs/#getcoord">Turf getCoord documentation</a>
-   * @since 1.2.0
-   */
-  public static Point getCoord(Feature obj) {
```

Comment:
This is a semver change, you need to deprecate and point to using the other method instead of deleting it here.

Reply:
@cammace this exactly what I was not sure about.
This  issue indeed said: deprecate but then 
https://github.com/mapbox/mapbox-java/pull/738
contained name changes and you reviewed it.
Some of those name changes were for API's since 1.2

Should I deprecate those as well instead of changename

Reply:
Yes, those need to be fixed also. Sorry for missing that in #738